hack/build-image

```diff
@@ -265,8 +265,8 @@ def container_tag(cli, target, tag, *tags):
     ]
     if "docker" not in base_args[0]:
         # podman can do it in one command, docker (on github ci) can not
-        args += [tag] + list(tags)
-        run(cli, args, check=True)
+        base_args += [tag] + list(tags)
+        run(cli, base_args, check=True)
```

Comment:
semantic quibble: base_args was meant to imply a starting set of arguments and then args was the complete set. Obviously, this was not implemented correctly but I'd prefer to either rename `base_args` as just `args` everywhere in this function or do something like: `args = base_args + [tag] + list(tags)`

Reply:
Done.